easybuild/easyconfigs/x/Xmipp/Xmipp-3.1-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Xmipp'
+version = '3.1'
+
+homepage = 'http://xmipp.cnb.csic.es/'
+description = "Xmipp is a suite of image processing programs, primarily aimed at single-particle 3D electron microscopy."
+
+source_urls = ['http://xmipp.cnb.csic.es/Downloads/']
+sources = ['Xmipp-%(version)s-src.tar.gz']
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+patches = ['sqlite_imports.patch', 'library_imports.patch']
+
+dependencies = [
+    ('Python', '2.7.9'),  # Make sure Python has a working SQlite in it, the install step creates a sqlite db.
```

Comment:
you should somehow check for this in the easyblock, rather than just mentioning this in a comment that nobody will read